src/docs/asciidoc/common/prerequisites/prerequisites.adoc

```diff
@@ -1,20 +1,12 @@
 :relative-path: ../../
 include::{docdir}/variables.adoc[]
 
-To create an application with ZetaPush you only need the Node.js ecosystem: 
+To create a ZetaPush application you only need the Node.js ecosystem: 
 
 [NOTE]
 ====
-- Node.js version 8.11.2 (LTS) or higher
-- npm version 5.x or higher
-====
-
-Node.js is used to:
-[NOTE]
-====
-- create custom cloud services
-- locally run custom cloud services
-- deploy your application on ZetaPush
+- Node.js version 10.3.0 (LTS) or higher
```

Comment:
I think we should keep here a link to use with npm 5.
Like this, I think it means we don't support neither npm 5 nor Node<10.3.0

Reply:
+1

Reply:
There no way to set Node version for the pushed context (Dockerfile) so it can problematic if a develop use Node Api no longer compatible in Node 10